custom_components/tuya_local/devices/ekf_connect_pro_wifi_type_c_powerstrip.yaml

```diff
@@ -0,0 +1,241 @@
+  - entity: switch
+    name: USB switch
+    class: switch
+    dps:
+      - id: 5
+        type: boolean
+        name: switch
+  - entity: number
+    category: config
+    name: Timer 1
+    icon: "mdi:timer"
+    dps:
+      - id: 9
+        name: value
+        type: integer
+        unit: min
+        range:
+          min: 0
+          max: 86400
+        mapping:
+          - scale: 60
+            step: 60
+  - entity: number
+    category: config
+    name: Timer 2
+    icon: "mdi:timer"
+    dps:
+      - id: 10
+        name: value
+        type: integer
+        unit: min
+        range:
+          min: 0
+          max: 86400
+        mapping:
+          - scale: 60
+            step: 60
+  - entity: number
+    category: config
+    name: Timer 3
+    icon: "mdi:timer"
+    dps:
+      - id: 11
+        name: value
+        type: integer
+        unit: min
+        range:
+          min: 0
+          max: 86400
+        mapping:
+          - scale: 60
+            step: 60
+  - entity: number
+    category: config
+    name: Timer 4
+    icon: "mdi:timer"
+    dps:
+      - id: 12
+        name: value
+        type: integer
+        unit: min
+        range:
+          min: 0
+          max: 86400
+        mapping:
+          - scale: 60
+            step: 60
+  - entity: number
+    category: config
+    name: USB timer
+    icon: "mdi:timer"
+    dps:
+      - id: 13
+        name: value
+        type: integer
+        unit: min
+        range:
+          min: 0
+          max: 86400
+        mapping:
+          - scale: 60
+            step: 60
+  - entity: sensor
+    category: diagnostic
+    name: Energy
+    dps:
+      - id: 17
+        name: sensor
+        type: integer
+        unit: Wh
+        readonly: true
+      - id: 25
+        type: integer
+        name: calibration
+        readonly: true
+  - entity: sensor
+    category: diagnostic
+    class: current
+    dps:
+      - id: 18
+        name: sensor
+        type: integer
+        class: measurement
+        readonly: true
+        force: true
+        unit: mA
+      - id: 23
+        type: integer
+        name: calibration
+        readonly: true
+  - entity: sensor
+    category: diagnostic
+    class: power
+    dps:
+      - id: 19
+        name: sensor
+        type: integer
+        class: measurement
+        readonly: true
+        force: true
+        unit: W
+        mapping:
+          - scale: 10
+      - id: 24
+        type: integer
+        name: calibration
+        readonly: true
+  - entity: sensor
+    category: diagnostic
+    class: voltage
+    dps:
+      - id: 20
+        name: sensor
+        type: integer
+        class: measurement
+        readonly: true
+        force: true
+        unit: V
+        mapping:
+          - scale: 10
+      - id: 22
+        type: integer
+        name: calibration
+        readonly: true
+  - entity: binary_sensor
+    class: problem
+    category: diagnostic
+    name: Test Bit
```

Comment:
I don't think this needs to be a sensor.  In shipped devices it is always 1.  In any case, the binary_sensor needs to have a dp with name "sensor", the name "factory_test" is suitable if it is an extra attribute on some other entity.

Reply:
Then I would remove it if it's not usable. I tried to make this as close as possible to other powerstrip configuration files.